### DISCUSSION TITLE
Multiple meters. Can I specify I only want smart meter data returned?

### DISCUSSION THREAD

**Discussion by pricei** [BottlecapDave/HomeAssistant-OctopusEnergy#501] [Q&A] [answered]
My Octopus account has three electricity meters associated with it. One is my current smart meter and the other two are old, non-smart meters that are no longer in use. I'm assuming because the Octopus API returns all three serial numbers for my MPAN, three separate devices (and sets of entities) are created by the integration. Is there a configuration option in the Octopus Energy integration that would allow me to specify that I only want data from the smart meter to be returned/collected?
Ian

**Comment by BottlecapDave** [comment]
The integration only create sensors for meters with active tariffs associated with them. So if you have old meters, they shouldn't be included as they shouldn't have an active tariff associated with them

**Reply by pricei** [reply]
Thanks for the quick reply, Dave. I checked and, although both my old meters have the 'Is smart meter' attribute set to false, they both have the Tariff attribute set to 'E-1R-VAR-22-11-01-E' which is presumably why a device and entitity collection has been created for them. I guess I need to ask Octopus to either delete the old meter serial numbers from my account or at least remove the active tariff associated with them? 
Ian

**Comment by pcb1962** [comment]
Can't you disable the devices you don't want? 
Click on the device in the Devices list, click the pen icon at the top right, uncheck Enable device, this should remove the device and all the associated entities.

**Reply by pricei** [reply]
That's a good suggestion. I've actually emailed Octopus to see if they can remove the unused meters from my account. If that can't be done, I'll try what you suggest. Thanks.
Ian

**Reply by pricei** [reply, marked as answer]
Just to close this one down, Octopus were very helpful and removed the two old meters from my account. I then deleted and re-installed the Octopus integration and my smart meter is now the only device showing. 
Ian
